ui/src/main/java/edu/wpi/grip/ui/codegeneration/data/TSocket.java

```diff
@@ -0,0 +1,53 @@
+package edu.wpi.grip.ui.codegeneration.data;
+
+
+public class TSocket {
+  protected String type;
```

Comment:
Why is the type being represented as a string instead of as the actual type that it is?